modules/rosa-sts-overview-of-the-default-cluster-specifications.adoc

```diff
@@ -23,21 +39,23 @@ You can quickly create a {product-title} (ROSA) cluster with the AWS Security To
 |* Default cluster version: Latest
 * Default AWS region for installations using the {cluster-manager-first} {hybrid-console-second}: us-east-1 (US East, North Virginia)
 * Default AWS region for installations using the `rosa` CLI: Defined by your `aws` CLI configuration
-* Availability: Single zone
+* Availability: Single zone for the data plane
 * Monitoring for user-defined projects: Enabled
 
 |Encryption
 |* Cloud storage is encrypted at rest
 * Additional etcd encryption is not enabled
 * The default AWS Key Management Service (KMS) key is used as the encryption key for persistent data
 
+ifndef::rosa-hcp[]
 |Control plane node configuration
 |* Control plane node instance type: m5.2xlarge (8 vCPU, 32 GiB RAM)
 * Control plane node count: 3
 
 |Infrastructure node configuration
 |* Infrastructure node instance type: r5.xlarge (4 vCPU, 32 GiB RAM)
 * Infrastructure node count: 2
+endif::rosa-hcp[]
```

Comment:
|Networking configuration
|* Cluster privacy: Public
* A new VPC is created for your cluster
@EricPonvelle The hosted cluster must be created with byo VPC. So there is not new VPC created for cluster
(Since I can't add comment on it and add it on this)

Reply:
Updated to 
![image](https://user-images.githubusercontent.com/16167833/221967357-df7953b0-5ac3-4b84-986f-d9a322dbf7c7.png)


@wgordon17 does this change work?